python/cudf/cudf/testing/_utils.py

```diff
@@ -330,3 +331,9 @@ def does_not_raise():
 
 def xfail_param(param, **kwargs):
     return pytest.param(param, marks=pytest.mark.xfail(**kwargs))
+
+
+numeric_dtypes_pairwise = pytest.mark.parametrize(
```

Comment:
I like this idea a lot. I would prefer a name indicating that it's a `parametrize` call, so you know the decorator is expected to fill in some parameters (rather than do something that modifies the test itself to be "pairwise" in some other way).
```suggestion
parametrize_numeric_dtypes_pairwise = pytest.mark.parametrize(
```